docs/feature/storage-backends/helm.mdx

```diff
@@ -198,20 +198,34 @@ capact act create cap.interface.helm.storage.install --name helm-storage -n capa
 1. Get output TypeInstances:
 
     ```bash
-    capact act get helm-storage -n capact-system -ojson | jq '.Actions[0].output.typeInstances'
+    capact act get helm-storage -n capact-system -ojsonpath -t '{.Actions[0].output.typeInstances}'
```

Comment:
I don't know why but I think that numerating of points does not work as intended. There are a couple of `1` instead of the next numbers.  Could you check this also? I verify that it does not work also in master, but perhaps it is an easy change.

Reply:
Fixed. I had to add additional space toe every line within these points.